### DISCUSSION TITLE
Possible to keep language server running after Helix exits, and re-connect on start?

### DISCUSSION THREAD

**Discussion by bonsairobo** [helix-editor/helix#8552] [Q&A]
For large projects, there is a significant delay from starting Helix until the language server is ready to respond to symbol queries. I think this could be essentially fixed by allowing the language server to continue running outside of the Helix process tree, but I don't know if this is possible or if there is any existing effort to make this possible.

**Comment by kirawi** [comment]
You can have it do something similar to https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#gdscript

**Reply by CptPotato** [reply]
For rust there's also [ra-multiplex](https://github.com/pr2502/ra-multiplex).
